src/transformers/trainer.py

```diff
@@ -3225,7 +3282,7 @@ def training_step(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor,
             with amp.scale_loss(loss, self.optimizer) as scaled_loss:
                 scaled_loss.backward()
         else:
-            self.accelerator.backward(loss)
+            self.accelerator.backward(loss, **kwargs)
```

Comment:
What happens if we pass the learning rate through when lomo isn't being used?

Reply:
It will break .. 😢  but we:
1- raise an error if users do not have the correct accelerate version with init-ing the trainer with lomo
2- pass `learning_rate` only if the optimizer is a lomo optimizer
3- removed `kwargs` in training step
So hopefully this should be safe enough 🙏